charts/incubator/lldap/templates/_secrets.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* Define the secrets */}}
+{{- define "lldap.secrets" -}}
+{{- $basename := include "tc.v1.common.lib.chart.names.fullname" $ -}}
+{{- $fetchname := printf "%s-lldap.secrets" $basename -}}
```

Comment:
Afaik this is incorrect, no idea where you got the idea that `-lldap` needed to be added here...